version.env

```diff
@@ -1,6 +1,6 @@
-CAPTURE_DATE=2022-07-01
-CAPTURE_DATE_PREV=2022-01-01
-DOB_DATA_DATE=20220705
-VERSION=22Q2
-VERSION_PREV=21Q2
-VERSION_GEO=22.2.2
+CAPTURE_DATE=
```

Comment:
See the comments on the issue template for the correct dates for the `CAPTURE_DATE`,  `CAPTURE_DATE_PREV`, and `DOB_DATA_DATE`